src/api/trips_v2.ts

```diff
@@ -31,8 +22,31 @@ export async function tripsSearch(
     when: query.when,
   };
 
-  const response = await client.post<TripsQuery>(url, cleanQuery, {
+  const data = await post<TripsQuery>(url, cleanQuery, opts);
+  return data;
+}
+
+export async function singleTripSearch(
+  queryString: string,
+  opts?: AxiosRequestConfig,
+): Promise<TripsQuery> {
+  const url = '/bff/v2/singleTrip';
+  const query = {
+    compressedQuery: queryString,
+  };
+  const data = await post<TripsQuery>(url, query, opts);
+  return data;
+}
+
+async function post<T>(
+  url: string,
+  query: any,
+  opts?: AxiosRequestConfig<any>,
+) {
+  const response = await client.post<T>(url, query, {
     ...opts,
+    // baseURL: 'http://10.0.2.2:8080', // uncomment for local BFF
```

Comment:
Er ikke dette kun for Android?

Reply:
jo, det stemmer. finnes det tilsvarende for iphone?